api.planx.uk/saveAndReturn/utils.ts

```diff
@@ -157,7 +157,7 @@ const validateSingleSessionRequest = async (
   try {
     const query = gql`
       query ValidateSingleSessionRequest($sessionId: uuid!) {
-        lowcal_sessions(where: { id: { _eq: $sessionId } }, limit: 1) {
+        lowcal_sessions_by_pk(id: $sessionId) {
```

Comment:
this is just minor cleanup while trying to narrow down where this bug was coming from - `_by_pk` implies `limit: 1` since it's the unique, primary key on our table